akka-actor-typed/src/main/scala/akka/actor/typed/Scheduler.scala

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (C) 2009-2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.actor.typed
+
+import akka.actor.Cancellable
+import akka.annotation.DoNotInherit
+
+import scala.concurrent.ExecutionContext
+import scala.concurrent.duration.FiniteDuration
+
+/**
+ * The ActorSystem facility for scheduling tasks.
+ *
+ * For scheduling within actors `Behaviors.withTimers` should be preferred.
+ *
+ * Not for user extension
+ */
+@DoNotInherit
+trait Scheduler {
+
```

Comment:
skipping sending to an actor from outside for now?

Reply:
Yep, only internal need is these two, for other cases probably better to tie to an actor lifecycle anyways.